pagopa/io-app
# fix: [IOBP-47] Small ui regression in idpay unsubscription flow

## Short description
fixed a small ui regression that  displayed a button in the wrong color
| before | after |
|--------|--------|
| <img width="402" alt="Screenshot 2023-05-17 at 17 16 11" src="https://github.com/pagopa/io-app/assets/60693085/c710a312-baf3-4f04-a989-a6cdb8186f1a"> |  <img width="512" alt="Screenshot 2023-05-18 at 11 36 27" src="https://github.com/pagopa/io-app/assets/60693085/3152893e-c1d2-4640-84bc-9c46987eb099">| 


## List of changes proposed in this pull request
- addition of `labelColor` prop to the corresponding `FooterWithButtons` component
## How to test
in an initiative's removal flow, make sure that the confirm button renders correctly.
